docs/README.md

```diff
@@ -46,6 +47,7 @@ Following sections are Kubernetes specific. If you are Kubernetes user, use foll
 | AWS EBS CSI Driver \ Kubernetes Version| v1.12 | v1.13 | v1.14 | v1.15 | v1.16 |
```

Comment:
I left this as is, but I can update the supported versions if needed.

Reply:
columns for 1.17 and 1.18 wont hurt, can update in other pr